.shellcheckrc

```diff
@@ -0,0 +1 @@
+external-sources=true
```

Comment:
Why?  Seems to me like any reasonably mature system will have some shell scripts sourcing one another.

Reply:
But it also enables `source`'ing any arbitrary file in your filesystem and `shellcheck` will follow.
I'm rather a fan of passing all the `.sh` files to the `shellcheck`, rather than enabling that

Reply:
Ah ok, I misread the docs for this.  No I agree with you, we don't want to shellcheck anything that gets sourced, will fix.